packages/wdio-junit-reporter/src/index.ts

```diff
@@ -201,7 +201,7 @@ class JunitReporter extends WDIOReporter {
                         // default
                         testCase.error(test.error.message)
                     }
-                    testCase.standardError(`\n${test.error.stack}\n`)
+                    testCase.standardError(`\n${test.error.stack?.replace(ansiRegex, '')}\n`)
```

Comment:
I don't know where these ASCII characters are coming from but I think we might have to apply this to the output on line 212 as well, what do you think?

Reply:
indeed, I'll do that

Reply:
@erwinheitzman finally it's not so trivial since this output is processed from objects and stringified by `JSON.stringify` which escaped back slashes and so the regex does not match escaped ansi codes. In the end I think it's not necessary to strip ansi codes from this output, because it seems to be mostly wdio internal logs (such as commands), and if a command send some ansi codes, I guess it's fine to log them.